/ch1w_back/lslidar_driver_node:
  ros__parameters:
    lidar_type: CH1W                    # 雷达类型
    device_ip: 192.168.1.203            # 雷达IP
    msop_port: 2373                     # 雷达目的数据端口
    difop_port: 2374                    # 雷达目的设备端口
    pcl_type: false                     # 发布点云类型 true: xyzi
    add_multicast: false                # 雷达是否为组播模式
    group_ip: 224.1.1.2                 # 组播IP
    use_time_service: false             # 雷达是否使用GPS/PTP/NTP授时
    min_range: 0.20                     # 雷达扫描最小距离,小于此值的点将被驱动过滤
    max_range: 500.0                    # 雷达扫描最大距离,大于此值的点将被驱动过滤
    scan_start_angle: 0                 # 雷达扫描最小角度,小于此值的点将被驱动过滤 单位: 0.01°
    scan_end_angle: 18000               # 雷达扫描最大角度,大于此值的点将被驱动过滤 单位: 0.01°
    frame_id: laser_link                # 雷达点云帧ID
    topic_name: lslidar_point_cloud     # 雷达点云话题名称
    echo_num: 0                         # 仅双回波模式下有效，0表示发布所有点云，1表示发布第一次回波点云，2表示发布第二次回波点云
    publish_scan: false                 # 驱动是否发布laserscan
    horizontal_angle_resolution: 0.024  # laserscan 角度分辨率 20hz: 0.012  80hz: 0.048
    packet_rate: 3510.0                 # 离线加载PCAP包,每秒PCAP包数
    #pcap: /home/ls/data/xxx.pcap       # PCAP包路径，加载PCAP包时打开此注释
